__tests__/components/AvatarView.test.tsx

```diff
@@ -2,9 +2,12 @@ import { render } from '@testing-library/react-native'
 import React from 'react'
 
 import AvatarView from '../../App/components/misc/AvatarView'
+import * as themeContext from '../../App/utils/themeContext' // note we're importing with a * to import all the exports
+import { defaultTheme } from '../../App/utils/themeContext'
 
 describe('AvatarView Component', () => {
   test('Renders correctly', async () => {
+    jest.spyOn(themeContext, 'useThemeContext').mockImplementation(() => defaultTheme)
```

Comment:
Would it be easier to add a global mock in `__mock__`?

Reply:
I didn't want to mock the whole `themeContext.ts` component, only that particular function. I am not sure how to that using `__mock__`. I previously had in a [setup file](https://jestjs.io/docs/configuration#setupfiles-array) , but I didn't want to introduce that just yet on this PR

Reply:
Also, `__tests__` are currently being ignored from typescript validation because there are a bunch of false positive errors. I think we will need to review and address  those those errors (possibly refactor) as another work.

Reply:
Ok. We can leave it for now and I can show you some mocking tricks. I think you'll create a file in `__mock__`` and use  `jest.requireActual()` or something then just mock what you want - not the entire thing.